uint8[] ids
float64[] positions       # rad
float64[] vel_profile   # モータの速度制限を変更 radian / sec
float64[] velocities    # radian
int64[] current
bool[]  torque_on
bool[]  reboot
int64[] errors